### PR TITLE
Fabric 6: Updating comments of text color semantic slots.

### PR DESCRIPTION
**Pull request by aneeshack4**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Updating comments of errorText, warningText, and successText.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9520)

### CODE REVIEW

File: packages/styling/src/interfaces/ISemanticTextColors.ts [hunk 1]
```diff
@@ -64,17 +64,17 @@ export interface ISemanticTextColors {
   //// Invariants - slots that rarely change color theme-to-theme because the color has meaning
 
   /**
-   * The default color of text indicating error on bodyBackground.
+   * The default color of text on errorBackground.
```
Comment (phkuo) [inline packages/styling/src/interfaces/ISemanticTextColors.ts]: No this is wrong, the original is correct.

Comment (aneeshack4) [reply packages/styling/src/interfaces/ISemanticTextColors.ts]: oops, thanks for catching this

File: packages/styling/src/interfaces/ISemanticTextColors.ts [hunk 2]
```diff
@@ -64,7 +64,7 @@ export interface ISemanticTextColors {
   //// Invariants - slots that rarely change color theme-to-theme because the color has meaning
 
   /**
-   * The default color of text on errorBackground.
+   * The default color of text on bodyBackground.
```
Comment (phkuo) [inline packages/styling/src/interfaces/ISemanticTextColors.ts]: default color of ERROR text

Comment (natalieethell) [reply packages/styling/src/interfaces/ISemanticTextColors.ts]:
```suggestion
   * The default color of error text on bodyBackground.
```

File: packages/styling/src/interfaces/ISemanticTextColors.ts [hunk 3]
```diff
@@ -64,17 +64,17 @@ export interface ISemanticTextColors {
...
 
   /**
-   * The default color of text indicating error on bodyBackground.
+   * The default color of error text on bodyBackground.
    */
   errorText: string;
 
   /**
-   * The color of text indicating warning on errorBackground, warningBackground, blockingBackground, or successBackground.
+   * The default color of warning text on errorBackground, warningBackground, blockingBackground, or successBackground.
```
Comment (phkuo) [inline packages/styling/src/interfaces/ISemanticTextColors.ts]: same comment as before here, this isn't the color of warning text, it's just the default color of text altogether on these backgrounds

Comment (aneeshack4) [reply packages/styling/src/interfaces/ISemanticTextColors.ts]: Ah I thought it was consistent with the other three text colors - like errorText is the color of error text so I thought warningText is the color of warning text too...

Comment (phkuo) [reply packages/styling/src/interfaces/ISemanticTextColors.ts]: Yeah it's confusing.
1) errorText is just the dark red we use for normal error text on normal bodyBackground. The name is old and predates Fabric.
2) warningText is new, and designed to work as the default color of text on top of a variety of backgrounds. Main scenario is message bar, but we use it in lists too. The background color indicates whether the message is a warning, error, or merely informational.

**Pull request merged**